src/selector/rbuggyQSA.js

```diff
@@ -23,6 +25,18 @@ if ( isIE ) {
 	rbuggyQSA.push( ":enabled", ":disabled" );
 }
 
+// Support: IE 11+, Edge 15 - 18+
+// IE 11/Edge don't find elements on a `[name='']` query in some cases.
+// Adding a temporary attribute to the document before the selection works
+// around the issue.
+// Interestingly, IE 10 & older don't seem to have the issue.
+input.setAttribute( "name", "" );
+testEl.appendChild( input );
+if ( !testEl.querySelectorAll( "[name='']" ).length ) {
+	rbuggyQSA.push( "\\[" + whitespace + "*name" + whitespace + "*=" +
```

Comment:
Does the `=` here also need to cover other operators (e.g., `[*^$]?=`)? Cf. checks for [empty strings after ^= or $= or *=](https://github.com/jquery/sizzle/blob/a528ae520245b13a6a879041e4a2b135c7d6e522/src/sizzle.js#L816) and [case-sensitivity of name attribute](https://github.com/jquery/sizzle/blob/a528ae520245b13a6a879041e4a2b135c7d6e522/src/sizzle.js#L858).

Reply:
Thanks, I'll have a look. Unfortunately, I merged the Sizzle PR before seeing this comment so it might need a followup PR later.

Reply:
@gibson042 What's the meaning of `!` inside of a regex there? Native CSS selectors don't support negated attribute selector `[name!=value]` so they'd go the Sizzle route anyway.

Reply:
> @gibson042 What's the meaning of `!` inside of a regex there? Native CSS selectors don't support negated attribute selector `[name!=value]` so they'd go the Sizzle route anyway.

Looks like it's there to make a longer repeat of https://github.com/jquery/sizzle/blob/a528ae520245b13a6a879041e4a2b135c7d6e522/src/sizzle.js#L87 for gzip.

Reply:
@gibson042 I checked the selectors you mentioned, the issue is all of them (`*=`, `^=`, `$=`) require the matched string to be non-empty to return any result. Therefore, all browsers return no results for them so there's nothing to workaround for IE here.